app/serializers/user_role_serializer.rb

```diff
@@ -1,3 +1,4 @@
 class UserRoleSerializer < ActiveModel::Serializer
   attributes :roles
+  belongs_to :user
```

Comment:
@vamshikrishnaprasthana  remove this change, it is already added by someone else.